st2common/st2common/constants/error_messages.py

```diff
@@ -32,3 +33,8 @@
 virtual environment with python2 binary:
 "st2 run packs.setup_virtualenv packs=%(pack)s python3=false"
 '''
+
+PYTHON2_DEPRECATION = 'DEPRECATION WARNING. Support for python 2 will be removed in future ' \
+                      'StackStorm release(s). Please ensure that all packs used are python ' \
+                      '3 compatible. Python 3 will already be used if you upgrade to a newer ' \
+                      ' OS release.'
```

Comment:
> Python 3 will already be used if you upgrade to a newer OS release.

I think this may sound a bit misleading due to generic message.

Maybe we could clarify it a bit? Something along the lines:
> Your StackStorm installation may be upgraded from python 2 to python 3 in the future platform releases. It's recommended to plan the manual migration to the python 3 native platform like Ubuntu 18.04 LTS or CentOS/RHEL 8.

WDYT? Feel free to adjust.

Reply:
Thanks for feedback, have made a change. What about following, or is too long?

`logs/st2workflowengine.log:2020-09-17 21:22:08,968 140485222490672 WARNING service_setup [-] DEPRECATION WARNING. Support for python 2 will be removed in future StackStorm releases. Please ensure that all packs used are python 3 compatible. Your StackStorm installation may be upgraded from python 2 to python 3 in future platform releases. It is recommended to plan the manual migration to a python 3 native platform, e.g. Ubuntu 18.04 LTS or CentOS/RHEL 8.`

Reply:
Looks good :+1: